core/common/src/main/java/alluxio/Configuration.java

```diff
@@ -69,8 +69,9 @@
   private static final String REGEX_STRING = "(\\$\\{([^{}]*)\\})";
   /** Regex to find ${key} for variable substitution. */
   private static final Pattern CONF_REGEX = Pattern.compile(REGEX_STRING);
-  /** Set of properties. */
-  private static final Properties PROPERTIES = new Properties();
+  /** Map of properties. */
+  private static final ConcurrentHashMapV8<String, String> PROPERTIES_MAP =
```

Comment:
Hi, Bin. I use this name is to distinguish the new one from previous one so as to avoid some weird error such as some coursed by whitebox get internal variable.